cypress/integration/sGMR/edit-vessel.spec.js

```diff
@@ -0,0 +1,64 @@
+describe('Edit existing vessel information', () => {
+  let vessel;
+
+  before(() => {
+    cy.registerUser();
+    cy.login();
+
+    cy.getVesselObj().then((vesselObj) => {
+      vessel = vesselObj;
+      cy.addVessel(vessel);
+    });
+  });
+
+  beforeEach(() => {
+    cy.login();
+    cy.navigation('Vessels');
+    cy.url().should('include', '/vessels');
+  });
+
+  it('Should be able to edit existing vessel information and save', () => {
+    cy.get('.govuk-table td a').contains(vessel.name).click();
```

Comment:
Alternatively, you could just use `contains`:
```suggestion
cy.contains('a', vessel.name).click();
```

Reply:
Done.